mountpoint-s3/src/fs.rs

```diff
@@ -666,6 +666,7 @@ where
                 None => return reply.error(err!(libc::EBADF, "invalid file handle")),
             }
         };
+        Span::current().record("name", handle.inode.name());
```

Comment:
Should we refactor all these calls into a function? Maybe in the `logging` module?
So that the call here would look something like (assuming we do want the key instead of the name):
```suggestion
        logging::record_key(&handle.full_key);
```